[Bug]: Compile Milvus with Cmake error

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:e66537f
- Deployment mode(standalone or cluster): both
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

```
CMake Error at src/index/thirdparty/CMakeLists.txt:108:
  Parse error.  Expected a command name, got unquoted argument with text
  "//Notice:".
```

### Expected Behavior

Compile successfully

### Steps To Reproduce

```markdown
1. clean local compile cache 
2. make all
```


### Anything else?

_No response_

--- a/internal/core/src/index/thirdparty/CMakeLists.txt
+++ b/internal/core/src/index/thirdparty/CMakeLists.txt
@@ -105,7 +105,7 @@ macro(build_faiss)
 
     message(STATUS "Building FAISS with configure args -${FAISS_CONFIGURE_ARGS}")
 	
-	//Notice: there will not set LDFLAGS value
+    #Notice: there will not set LDFLAGS value
     if (DEFINED ENV{FAISS_SOURCE_URL})
         set(FAISS_SOURCE_URL "$ENV{FAISS_SOURCE_URL}")
         externalproject_add(faiss_ep

